dist/example_config.yaml

```diff
@@ -164,6 +163,34 @@ chihaya:
   #     # The timeout for connecting to redis server.
   #     redis_connect_timeout: "15s"
 
+  # This block defines configuration used for database storage.
+  # storage:
+  #   name: "sqlite" # "postgres" and "sqlite" currently supported.
+  #   config:
+  #     # The frequency which stale peers are removed.
+  #     # This balances between
+  #     # - collecting garbage more often, potentially using more CPU time, but potentially using less memory (lower value)
+  #     # - collecting garbage less frequently, saving CPU time, but keeping old peers long, thus using more memory (higher value).
+  #     gc_interval: "3m"
+
+  #     # The amount of time until a peer is considered stale.
+  #     # To avoid churn, keep this slightly larger than `announce_interval`
+  #     peer_lifetime: "31m"
+
+  #     # The interval at which metrics about the number of infohashes and peers
+  #     # are collected and posted to Prometheus. Disabled by default.
+  #     prometheus_reporting_interval: "1s"
+
+  #     # In lightweight mode, the announces and scrapes pretends there are no peers in the swarm.
```

Comment:
From the implementation, this seems to only apply to scrapes (which I think is correct). If that is indeed intended, maybe rename this to `ignore_scrapes` or something like that?